### PR TITLE
Update influxdb vendoring

### PR DESCRIPTION
**Pull request by SuperQ**
* Update vendor github.com/influxdata/influxdb/...
* Updates to interface with influxdb model.

See: https://github.com/influxdata/influxdb/pull/7138

### CODE REVIEW

**Comment by brian-brazil** [comment]
Thanks!

File: main.go [hunk 1]
```diff
@@ -138,7 +138,12 @@ func (c *influxDBCollector) influxDBPost(w http.ResponseWriter, r *http.Request)
 
 func (c *influxDBCollector) parsePointsToSample(points []models.Point) {
 	for _, s := range points {
-		for field, v := range s.Fields() {
+		fields, err := s.Fields()
+		if err != nil {
+			log.Errorf("error getting fields from point: %s", err)
```
Comment (juliusv) [inline main.go]: Start log messages (vs. error strings) upper-cased.

**Comment by juliusv** [comment]
:+1: besides nit

File: main.go (hunk 1 above)
Comment (SuperQ) [inline main.go]: Whups, merged, I'll fix in master.

**Pull request merged**